### DISCUSSION TITLE
Can commits on GitHub Desktop be marked "verified"?

### DISCUSSION THREAD

**Discussion by weeklyd3** [community/community#21606] [New to GitHub]
<p>Commits on GitHub Desktop are marked “unverified”:<br>
<img src="https://user-images.githubusercontent.com/79176077/181065806-9c458606-3318-4077-a09b-6dc68d182ba9.png" alt="image" data-base62-sha1="b3uJKKuIVeucEUHcNx9J7tgECRk" width="628" height="468"><br>
(the unverified commits are from Desktop)</p>
<p>So is there a way to make commits from GitHub Desktop “verified”?</p>

**Comment by loganbek** [comment, marked as answer]
<p>Unfortunately, according to the docs, Github Desktop does not support commit signing: <a href="https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/signing-commits" rel="noopener nofollow ugc">Signing Commits Docs</a></p>
